worker/worker.go

```diff
@@ -24,7 +24,7 @@ type EventHandlerFunc func(e ipc.DispatcherEvent)
 type Worker struct {
 	directive     string
 	features      map[string]string
-	remoteContent bool
+	RemoteContent bool
```

Comment:
Not sure about this change, but It is useful to be able to get this information from the worker. I know that it is possible to get this information using `GetFeature()`, but it makes sense to do it in `yggd`. Doing this in code of worker seems as overkill.